index.bs

```diff
@@ -2536,6 +2539,59 @@ To <dfn>get the navigation info</dfn>, given |context| and |navigation status|:
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.close Command ####  {#command-browsingContext-close}
+
+The <dfn export for=commands>browsingContext.close</dfn> command closes a
+[=top-level browsing context=] and returns the list of all top-level contexts.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextCloseCommand = {
+        method: "browsingContext.close",
+        params: BrowsingContextCloseParameters
+      }
+
+      BrowsingContextCloseParameters = {
+        context: BrowsingContext
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+        BrowsingContextCloseResult = {
+          contexts: BrowsingContextInfoList
+        }
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.close">
+The [=remote end steps=] with |command parameters| are:
+
+  1. Let |context id| be the value of the <code>context</code> field of
+     |command parameters|.
+
+  1. Let |context| be the result of [=trying=] to [=get a browsing context=]
+     with |context id|.
+
+  1. Assert: |context| is not null.
+
+  1. Assert: |context| is a [=top-level browsing context=]
+
+  <!-- XXX: This is step 2 from https://w3c.github.io/webdriver/#close-window,
+       but not sure if it needs more adaptation here? -->
+  1. [=Handle any user prompts=] and return its value if it is an [=error=].
+
+  1. [=Close=] |context|.
+
+  1. Return the result of running the [=remote end steps=] for the
```

Comment:
Arguably we don't need to do this since a client could just use the relevant events to keep track of which contexts still exist.

Reply:
I just removed any return value, is that ok?